website/docs/backends/types/remote.html.md

```diff
@@ -11,20 +11,20 @@ description: |-
 **Kind: Enhanced**
 
 -> **Note:** We recommend using Terraform v0.11.13 or newer with this
-backend. This backend requires either a Terraform Enterprise account on
-[app.terraform.io](https://app.terraform.io) or a private instance of
-Terraform Enterprise (version v201809-1 or newer).
+backend. This backend requires either a Terraform Cloud account on
+[app.terraform.io](https://app.terraform.io) or a Terraform Enterprise instance
+(version v201809-1 or newer).
 
-The remote backend stores state and runs operations in Terraform Enterprise.
+The remote backend stores state and runs operations in Terraform Cloud.
 
-When used with a Pro or Premium tier Terraform Enterprise account, operations
+When used with a Pro or Premium tier Terraform Cloud account, operations
 like `terraform plan` or `terraform apply` are executed in Terraform
 Enterprise's run environment, with log output streaming to the local terminal.
```

Comment:
should the next two `Enterprise`s also be changed to `Cloud`?

Reply:
😱 They should, but I missed them because HARD WRAP 😭🤬

Excellent example of why I hate hard wrap.

Reply:
Fixed!